tests/rados/monitor_configurations.py

```diff
@@ -374,3 +374,23 @@ def ceph_config_reset(self, version: int) -> bool:
         except Exception:
             log.error("The log collected does not contain the name of change made")
             return False
+
+    def show_config(self, daemon: str, id: str, param: str) -> str:
+        """
+        Reports the  running configuration value of a paramter for a running daemon
+        Usage - config show <daemon>.<id> <param>
+        Args:
+            daemon: Ceph daemons like osd,mon,mgr.
+            id: Id's of the mon
+            param: Parameter name
+        Returns: null if no output from the command or string output of ceph config shoe <daemon>.<id>  command.
+        E.g.
+            - # ceph config show osd.0 osd_recovery_max_active
+        """
+        cmd = f"ceph config show {daemon}.{id} {param}"
+        try:
+            result, _ = self.rados_obj.node.shell([cmd])
+        except Exception:
```

Comment:
The error msg for an invalid parameter will be part of STDERR and will not raise exception.
Please modify the code to check if `Error ENOENT:` is present in stderr

Reply:
Modified the code.